app/assets/javascripts/discourse/app/components/sidebar/section-link.hbs

```diff
@@ -7,7 +7,7 @@
       <a
         href={{@href}}
         rel="noopener noreferrer"
-        target={{or @target "_blank"}}
+        target="_blank"
```

Comment:
That was added in previous PR but once we moved to Ember route it is not required anymore https://github.com/discourse/discourse/pull/20057/files#diff-3f867149b0de3b82c2c5ddfee99564eee7b238eaf44775f4859da4677939d1a8R10